scripts/backCompatChecker.py

```diff
@@ -85,21 +93,27 @@ def findExtension():
             if file.endswith('.whl'):
                 return os.path.join(p, file)
 
+
 newExtensionLocation = findExtension()
+print('Install extension (loaded from current code). Wheel path - {}'.format(newExtensionLocation))
 subprocess.run(['az', 'extension', 'add', '--source', newExtensionLocation, '-y'], shell=True, stdout=subprocess.PIPE)
 
+# Check the installed extensions
+subprocess.run(['az', 'extension', 'list'], shell=True)
+
 # get a set of old commands, we are not reusing the set from ext because we want to keep this clean
 oldCommands = []
 for oldArgument in oldArguments:
     if oldArgument.command not in ignoreCommands:
-        if not any(oldArgument.command in s for s in oldCommands):
+        if not (oldArgument.command in oldCommands):
```

Comment:
FIX 3:  Substring match issue in list membership check. The check fails for command = 'devops security permission reset' list containing 'devops security permission reset-all' and thus all params of command are being detected as missing.